### PR TITLE
Improve Run Instances Dialog

### PR DESCRIPTION
**Pull request by KoBeWi**
Small improvements to the dialog:
- moved main arguments and features to be next to each other
- added placeholders to clarify usage
- squashed instance count

Before:
image

After:
image

### CODE REVIEW

**Comment by seppoday** [comment]
I guess this kind of design might work? 
image

**Comment by KoBeWi** [comment]
This makes the instance count disjointed from the list 🤔
Also the counter would be better on the left probably.

**Comment by Calinou** [comment]
I like @seppoday's mockup but I'd put the instance count on the left too.

**Comment by KoBeWi** [comment]
image

**Review by akien-mga — APPROVED** [review]
Looks good to me.

File: editor/run_instances_dialog.cpp [hunk 1]
```diff
@@ -271,52 +272,56 @@ RunInstancesDialog::RunInstancesDialog() {
...
+	main_args_edit->set_h_size_flags(Control::SIZE_EXPAND_FILL);
+	main_args_edit->set_placeholder(TTR("Space-separated arguments, e.g. host player1 blue"));
+	args_gc->add_child(main_args_edit);
+	_fetch_main_args();
+	ProjectSettings::get_singleton()->connect("settings_changed", callable_mp(this, &RunInstancesDialog::_fetch_main_args));
+	main_args_edit->connect("text_changed", callable_mp(this, &RunInstancesDialog::_start_main_timer).unbind(1));
+
+	main_features_edit = memnew(LineEdit);
+	main_features_edit->set_h_size_flags(Control::SIZE_EXPAND_FILL);
+	main_features_edit->set_placeholder(TTR("Space-separated tags, e.g. demo, steam, event"));
```
Comment (Calinou) [inline editor/run_instances_dialog.cpp:320]:
```suggestion
	main_features_edit->set_placeholder(TTR("Comma-separated tags, example: demo, steam, event"));
```

File: editor/run_instances_dialog.cpp [hunk 2]
```diff
@@ -271,52 +272,56 @@ RunInstancesDialog::RunInstancesDialog() {
...
-	main_vb->add_child(instance_count);
+	args_gc->add_child(instance_count);
 	instance_count->connect("value_changed", callable_mp(this, &RunInstancesDialog::_start_instance_timer).unbind(1));
 	instance_count->connect("value_changed", callable_mp(this, &RunInstancesDialog::_refresh_argument_count).unbind(1));
 	enable_multiple_instances_checkbox->connect("toggled", callable_mp(instance_count, &SpinBox::set_editable));
 	instance_count->set_editable(enable_multiple_instances_checkbox->is_pressed());
 
+	main_args_edit = memnew(LineEdit);
+	main_args_edit->set_h_size_flags(Control::SIZE_EXPAND_FILL);
+	main_args_edit->set_placeholder(TTR("Space-separated arguments, e.g. host player1 blue"));
```
Comment (Calinou) [inline editor/run_instances_dialog.cpp:312]:
```suggestion
	main_args_edit->set_placeholder(TTR("Space-separated arguments, example: host player1 blue"));
```

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**